Gruntfile.js

```diff
@@ -162,7 +162,17 @@ module.exports = function Gruntfile( grunt ) {
 		clean: [ 'dist/', 'temp/' ],
 		shell: {
 			mocha: 'mocha --require @babel/register test/testHelper.js --recursive test/suite  --colors',
-			webextBuild: 'web-ext build',
+			webextBuild: [
```

Comment:
OK I'm going to be a little petty, but this is really confusing and I want to make sure we know what this is doing even when we get back to this in 6 months :stuck_out_tongue_winking_eye: 

Let's document here that 
- The chromeManifest job _runs first_
- This specific job ends up with files in `dist/` and also 2 zip files for the extensions
- Optionally, emphasize that there's a difference between the manifest in `dev` and the zipped one. It doesn't matter in practice, but we should remember that when/if debugging something.

Reply:
Okay, I've added some more comments. I've also moved the source zip file into the `extension_firefox` directory, so it's not hanging out on its own in the directory above.